data/json/items/armor.json

```diff
@@ -4465,7 +4465,7 @@
         "volume" : 8,
         "warmth" : 25,
         "encumbrance" : 16,
-        "flags" : ["STURDY", "BLOCK_WHILE_WORN"],
```

Comment:
Reason stated above, it ceases to have `BLOCK_WHILE_WORN` when used to craft boiled or plated leather armor, while adding it to complete sets might be a bit odd. I can revert this change if desired.